spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -462,37 +461,6 @@ class CometExecSuite extends CometTestBase {
     }
   }
 
-  test("CometExec.executeColumnarCollectIterator can collect ColumnarBatch results") {
```

Comment:
This test was testing `executeColumnarCollectIterator` which was not used anywhere except for in this test and is removed as part of this PR.